roles/ceph-defaults/tasks/facts.yml

```diff
@@ -127,6 +127,13 @@
     - rbd_client_directory_mode is not defined
       or not rbd_client_directory_mode
 
+- name: set_fact devices to empty list
```

Comment:
why not something like this? https://github.com/ceph/ceph-ansible/pull/2000/files#diff-2857b72f37476e1b5d16e7e81783ecbaR137

Reply:
Well, this is matter of taste... and work time =)
The `devices` may use elsewhere (not only in `ceph-defaults`), so, since we explicitly define var, no further changes needed.